src/test/java/org/folio/logintest/LoginAttemptsTest.java

```diff
@@ -111,7 +111,7 @@ public static void setup(final TestContext context) throws Exception {
   public void setUp(TestContext context) {
     MODULE_SPECIFIC_ARGS.clear();
     MODULE_SPECIFIC_ARGS.putAll(moduleArgs);
-    UserMock.resetConfigs();
+    Mocks.resetConfigs();
```

Comment:
Semantically didn't like the name of this class since it contains mocks for a lot more than user stuff. So renamed it.